openapi-cli/src/test/resources/testng.xml

```diff
@@ -89,6 +89,7 @@ under the License.
             <class name="io.ballerina.openapi.generators.schema.AnnotationTests"/>
             <class name="io.ballerina.openapi.generators.testcases.BallerinaTestGeneratorTests"/>
             <class name="io.ballerina.openapi.generators.client.OneOfResponsesTests"/>
+            <class name="io.ballerina.openapi.generators.openapi.DataTypeTests"/>
```

Comment:
I've noticed a separate test class named `RecordDataTypeTests`. Do we need to have a separate class only for record types?

Reply:
Yeah in the `RecordDataTypeTests`  used to store the data related records and suppose to add `DataTypeTests` for maintaining the special scenarios of data type, it can be primitive, record, user-defined data types. :)